keps/sig-ui/5008-kep-template/kep.yaml

```diff
@@ -0,0 +1,37 @@
+title: Kubernetes Desktop
+kep-number: 5008
+authors:
+  - "@joaquimrocha"
+owning-sig: sig-ui
+participating-sigs:
+status: provisional
+creation-date: 2024-12-19
+reviewers:
+  - TBD
+approvers:
+  - TBD
+
+see-also:
+replaces:
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha|beta|stable
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v1.19"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
+  alpha: "v1.19"
+  beta: "v1.20"
+  stable: "v1.22"
```

Comment:
the metadata is wrong, I wonder if you really needed a KEP for this

Reply:
Right. Our KEP is not changing kubernetes's core nor is dependent/following a particular version.
I think I left the milestones like this because this section was needed? I am happy to remove it or adapt it as appropriate.

Reply:
The point is: does this need a KEP at all?

If it's only about contributing some source code to SIG UI, then talk to that SIG in their SIG meetings and then come back for repo creation following the process in https://github.com/kubernetes/community/blob/master/github-management/subproject-responsibilities.md

Reply:
We do have KEPs for changes that don't have milestones.

If we want to change project policy (example: we'd encourage SIGs to ship Headlamp plugins and to build for Headlamp), a KEP is useful.

If we're _only_ adopting a donated component, I don't see a need for a KEP.

---

I think we should propose making Headlamp (or some other name for the same component) official and endorsed. Personal opinion.

Reply:
I will remove the milestones.